public/docs/ts/latest/guide/architecture.jade

```diff
@@ -151,7 +148,7 @@ figure
 
 .l-sub-section
   :marked
-    Learn more in the [Angular Modules](ngmodule.html) chapter.
+    Learn more on the [Angular modules](ngmodule.html) page.
```

Comment:
"on" sounded weird to me. It sounded like I was jumping on the page. :)

Instead: "from"? "at"?